common/lib/xmodule/xmodule/course_module.py

```diff
@@ -940,6 +946,10 @@ def set_grading_policy(self, course_policy):
         self.raw_grader = grading_policy['GRADER']  # used for cms access
         self.grade_cutoffs = grading_policy['GRADE_CUTOFFS']
 
+    def set_default_certificate_available_date(self):
+        if (not self.certificate_available_date) and self.end:
+            self.certificate_available_date = self.end + timedelta(days=2)
```

Comment:
The easiest way is to just throw a constant in `lms/envs/common.py`, and let it be overriden by aws.py with a configuration value as needed.

Here are some searches showing how we did it for EDXAPP_RECALCULATE_GRADES_ROUTING_KEY:

- https://github.com/edx/edx-platform/search?&q=RECALCULATE_GRADES_ROUTING_KEY
- https://github.com/edx/configuration/search?utf8=%E2%9C%93&q=RECALCULATE_GRADES_ROUTING_KEY&type=